### ISSUE TITLE
Show flag after completion

### ISSUE THREAD

**Issue by stealthsploit** [moloch--/RootTheBox#206]
It would be useful if after successfully completing a flag, the flag itself could be displayed alongside when viewing the box. Or maybe a toggle button to show completed flags?

**Comment by jeffg2k** [comment]
Pushed update that adds a flag button to display the token after it has been answered.  I also expanded the area for flag description.  Made the area hidden after answered, but expandable.  Made the Attempts column dependant on the penalty option.  Added a captures column for the flags to show how many times that flag has been captured (which I found useful in assessing the difficulty for what flag I want to attempt next).
